cpp/src/gcylon/utils/util.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <glog/logging.h>
+#include <gcylon/utils/util.hpp>
+
+#include <cudf/sorting.hpp>
+#include <cudf/binaryop.hpp>
+#include <cudf/reduction.hpp>
+#include <cudf/aggregation.hpp>
+#include <cudf/scalar/scalar.hpp>
+
+namespace gcylon {
+
+bool equal(cudf::table_view & tv1, cudf::table_view & tv2) {
+    std::unique_ptr<cudf::table> sortedTable1 = cudf::sort(tv1);
+    auto sortedTv1 = sortedTable1->view();
+
+    std::unique_ptr<cudf::table> sortedTable2 = cudf::sort(tv2);
+    auto sortedTv2 = sortedTable2->view();
+
+    if (sortedTv1.num_columns() != sortedTv2.num_columns()){
+        LOG(INFO) << "number of columns do not match: " << sortedTv1.num_columns() << " != " << sortedTv2.num_columns();
```

Comment:
my suggestion is not to have these logs

Reply:
removed most of the logs and commented out statements